src/imagecapturecore.cs

```diff
@@ -495,7 +495,7 @@ interface IICCameraDeviceDownloadDelegate {}
 	interface ICCameraDeviceDownloadDelegate {
 
 		[Export ("didDownloadFile:error:options:contextInfo:")]
-		void DidDownloadFile (ICCameraFile file, [NullAllowed] NSError error, [NullAllowed] NSDictionary<NSString, NSObject> options, [NullAllowed] IntPtr contextInfo);
+		void DidDownloadFile (ICCameraFile file, [NullAllowed] NSError error, NSDictionary<NSString, NSObject> options, [NullAllowed] IntPtr contextInfo);
```

Comment:
This change and the two below remove a NullAllowed on NSDictionary parameters. The headers and docs both agree that these are not nullable - but wanted to make sure that removing these wouldn't be a breaking change?

Reply:
I don't believe this is a breaking change, though it could in theory break a build if:

- They have null allowed warnings on
- They have warnings as error enabled
- They were passing in null to this param.

But I'm fine with that I think.